### ISSUE TITLE
No Python in IDEA community edition

### ISSUE THREAD

**Issue by donkirkby** [donkirkby/live-py-plugin#532]
I'm pretty sure I originally used the community edition to create the PyCharm plugin, but I've been using the ultimate edition since JetBrains gave me free licenses to maintain this project.

Recently, I found that new contributors couldn't use the community edition, because it doesn't support the Python plugin.

Try to find a way for new contributors to run the plugin from source. If not, at least update the contributing file, and recommend the 30-day free trial.

**Comment by donkirkby** [comment]
OK, now I'm more confused. The [feature comparison] page says the community edition supports Python, and I just installed it successfully on Ubuntu. Is MacOS different?

I'll have to find someone to try it again on Mac.

[feature comparison]: https://www.jetbrains.com/products/compare/?product=idea&product=idea-ce

**Comment by mtkalms** [comment]
I have the latest version of IntelliJ IDEA CE (Build #IC-222.4554.10, built on March 14, 2023) installed on my Mac. There is a compatible version of the Python Community Edition plugin, which was installed together with the Live Coding in Python plugin. Everything works as expected. After configuring the Python SDK, I can run the Live Coding for Python plugin and even test current changes with runIde. 

In short, I did not encounter any issues in IDEA.

**Comment by donkirkby** [comment]
OK, it sounds like I was the one who got confused by the two different Python plugins.

Thanks for testing that out, @mtkalms. I'll update `CONTRIBUTING.md`, and close this issue.
